### PR TITLE
Deletes a duplicated sentence in Docker section

### PR DESCRIPTION
**Pull request by pc494**
Assumed to be a typo, couldn't see sensible context for it otherwise.

### Summary

I couldn't find an open issue or PR for this, and it seemed like a good first PR to raise.

### List of changes proposed in this PR (pull-request)

* [x] Deletes a duplicated sentence

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: @pc494

### CODE REVIEW

**Review by Arielle-Bennett — APPROVED** [review]
Looks OK - agree that it appears to be a duplicated sentence.

File: book/website/reproducible-research/renv/renv-containers.md [hunk 1]
```diff
@@ -376,7 +376,6 @@ If the second part (the `path_to_where_to_put_file/file_name`) is substituted fo
 ## Volumes
 
 Every time a container is opened from an image, that container is completely new.
-Say a container is opened, and work is done within it.
 Say a container is opened, and work is done within it. If that container is closed, and the image it came from is again used to start another container, none of that work will be in the new one.
```
Comment (da5nsy) [inline book/website/reproducible-research/renv/renv-containers.md:379]:
```suggestion
Say a container is opened, and work is done within it. 
If that container is closed, and the image it came from is again used to start another container, none of that work will be in the new one.
```
Adding a semantic line break (as per [here](https://the-turing-way.netlify.app/community-handbook/style.html#write-each-sentence-in-a-new-line-line-breaks))

**Review by da5nsy — CHANGES_REQUESTED** [review]

**Review by da5nsy — APPROVED** [review]

**Comment by da5nsy** [comment]
Thanks @pc494! 
I added a semantic line break (as per [here](https://the-turing-way.netlify.app/community-handbook/style.html#write-each-sentence-in-a-new-line-line-breaks)). Otherwise, all LGTM so I'll merge!

@all-contributors please add @pc494 for bug

**Comment by pc494** [comment]
> Thanks @pc494! I added a semantic line break (as per [here](https://the-turing-way.netlify.app/community-handbook/style.html#write-each-sentence-in-a-new-line-line-breaks)). Otherwise, all LGTM so I'll merge!
> 
> @all-contributors please add @pc494 for bug

Thank you, sorry for the error there.

**Pull request merged**